src/test/java/com/flowpowered/nbt/stream/EndianSwitchableStreamTest.java

```diff
@@ -44,9 +44,11 @@ public void testWriteLEUnsignedShort() throws IOException {
         EndianSwitchableOutputStream output = new EndianSwitchableOutputStream(rawOutput, ByteOrder.LITTLE_ENDIAN);
         output.writeShort(unsigned);
         output.writeChar(testChar);
+        output.close();
```

Comment:
// just a random comment, not important for this review
Kind of ironic that there are tests for LITTLE_ENDIAN short and char but not float and double. This would have saved me 3hrs of debugging. :'(
(see #16)